pkg/controllers/termination/eviction.go

```diff
@@ -103,6 +103,7 @@ func (e *EvictionQueue) evict(ctx context.Context, nn types.NamespacedName) bool
 		return true
 	}
 	if err != nil {
+		logging.FromContext(ctx).Debugf("Could not evict pod %s due to an unexpected error: %s", nn.String(), err)
```

Comment:
I think this should be ERROR level.

Reply:
Unless this is massively noisy. It only happens during misconfiguration, right?

Reply:
See the difference between lines 95 and 99

Reply:
Should the 429 violation of PDB be turned up to an info log as well?  It may save a round of "turn on debug logs" when troubleshooting why a node won't drain.

Reply:
Seems to me they all should be Error level.  "Error" logs will show up with the default log config, I believe.  

One note, we do retry eviction failures, so we will see multiple messages in the log file for each pod.

Reply:
The 429 seems like a different sort of transient condition to me. It may resolve itself if another pod becomes ready. It doesn't seem right for a log to have repeated error messages when everything is possibly working as expected. 

I agree that the multiple PDBs definitely seems like an error level condition. The user has to do something to resolve it.

Reply:
> The 429 seems like a different sort of transient condition to me. It may resolve itself if another pod becomes ready. It doesn't seem right for a log to have repeated error messages when everything is possibly working as expected.

This is exactly the reason. We want to avoid spamming INFO logs for normal operation. We're pretty careful about what meets the bar for info. I expect most customers will want to run with DEBUG on and then filter with different tooling. Perhaps we should make DEBUG the default log level in our helm chart.

Reply:
A 429 should just be debug log IMO. If you run into this error message once, you'll run into it at least 10 times per pod due to the fast exponential backoff, which would make the logs pretty noisy. 

A 500 I would think should be info level as well. An unexpected error would make sense to be info level as well.